lib/src/message_window.dart

```diff
@@ -14,32 +15,73 @@ const int _headerByteSize = 5;
 typedef _ServerMessageFn = ServerMessage Function(
     PgByteDataReader reader, int length);
 
-Map<int, _ServerMessageFn> _messageTypeMap = {
-  49: (_, __) => ParseCompleteMessage(),
-  50: (_, __) => BindCompleteMessage(),
-  65: (r, _) => NotificationResponseMessage.parse(r),
-  67: (r, _) => CommandCompleteMessage.parse(r),
-  68: (r, _) => DataRowMessage.parse(r),
-  69: ErrorResponseMessage.parse,
-  75: (r, _) => BackendKeyMessage.parse(r),
-  82: AuthenticationMessage.parse,
-  83: (r, l) => ParameterStatusMessage.parse(r),
-  84: (r, _) => RowDescriptionMessage.parse(r),
-  87: (r, _) => CopyBothResponseMessage.parse(r),
-  90: ReadyForQueryMessage.parse,
-  100: _parseCopyDataMessage,
-  110: (_, __) => NoDataMessage(),
-  116: (r, _) => ParameterDescriptionMessage.parse(r),
-  $3: (_, __) => CloseCompleteMessage(),
-  $N: NoticeMessage.parse,
-};
+// Map<int, _ServerMessageFn> _messageTypeMap = {
```

Comment:
Is there a reason for changing this to a switch statement? If all else being equal, let's not change this (or if this is needed, let's do this in a different PR).

Reply:
At first I thought I needed to change this but then I saw that it wouldn't be necessary, it turns out that I forgot to revert this change.